src/components/Avatar/index.tsx

```diff
@@ -73,16 +76,21 @@ const Avatar: FunctionComponent<AvatarProps> = ({
         textBgColor={textBgColor}
         textColor={textColor}
         textSize={textSize}
+        alt={lock ? 'locked' : text}
```

Comment:
I don't get it? `alt` on a `div`? 

Maybe `aria-label` and I'm not even sure?

Reply:
if it not possible to add title tag in svg (see my comment below), use `aria-label="locked"` and ONLY if it is lock